tephra-hbase-compat-0.98/src/main/java/co/cask/tephra/hbase98/coprocessor/TransactionProcessor.java

```diff
@@ -151,7 +151,7 @@ public void preGetOp(ObserverContext<RegionCoprocessorEnvironment> e, Get get, L
     if (tx != null) {
       projectFamilyDeletes(get);
       get.setMaxVersions();
-      get.setTimeRange(TxUtils.getOldestVisibleTimestamp(ttlByFamily, tx), TxUtils.getMaxVisibleTimestamp(tx));
+      get.setTimeRange(0, TxUtils.getMaxVisibleTimestamp(tx));
```

Comment:
I wonder, whether this has a performance penalty. With a lower bound, the scan might be able to optimize more, for example, prune the store files that need to be scanned (if a file has only timestamps that are outside the bounds). That pruning would now have to happen as a (post) filter while scanning the file. I can't prove that it would make a difference, but I have a feeling that it might.